lib/scrabble.rb

```diff
@@ -1,18 +1,57 @@
+require './word_finder'
+require './word_list'
+
 class Scrabble
+  attr_accessor :point_values
+
+  def initialize(word, score)
+    @word = word    
+    @score = 0
+    @point_values = ''
+    @letter = ''
+  end
```

Comment:
The interaction pattern doesn't pass any arguments to initialize. I wouldn't expect to see any parameters here. You could use the instance variables you have set here, but you shouldn't need to. I'd use local variables in the methods instead.